src/index.js

```diff
@@ -0,0 +1,50 @@
+import CarModel from './model/CarModel.js';
+import InputView from './view/InputView.js';
+
+function printOneGame(nameList, cars) {
+  for (let i = 0; i < nameList.length; i++) {
+    const carOutput = '-'.repeat(cars[i].position);
+    console.log(`${nameList[i]} : ${carOutput}`);
+  }
+  console.log('');
+}
+
+// 입출력 예시
+export async function run() {
+  const cars = [];
+
+  const nameList = await InputView.getNameList();
+  const count = await InputView.getCount();
+
+  nameList.forEach(name => {
+    const car = new CarModel(name);
+    cars.push(car);
+  });
+
+  console.log('\n실행 결과');
+  for (let i = 0; i < count; i++) {
+    for (let j = 0; j < nameList.length; j++) {
+      const currentCar = cars[j];
+      currentCar.go();
+    }
+    printOneGame(nameList, cars);
+  }
```

Comment:
다음과 같이 간소화할 수 있습니다.

```suggestion
  Array.from({ length: count }, () => {
    cars.forEach(car => car.go());
    printOneGame(nameList, cars);
  });
```
중첩된 for 문의 사용은 코드 가독성을 크게 낮춥니다. 최대한 지양해주세요.

Reply:
Array.from같은 문법을 활용할 생각을 못했었던게 반성이 됩니다... 가독성을 생각하며 고쳐보도록 하겠습니다! 감사합니다!

Reply:
개별질문에 대한 답변에 코멘트를 못달아서 여기에 적어요!

코드 하나하나 읽어주시고, 캡쳐까지 해주시면서 동작을 확인해주셔서 너무 감사합니다ㅠㅠ 제가 고민했던 부분을 납득할수있었던 여러 말씀들도 너무 감사드려요...! 프론트는 시간복잡도를 크게 신경쓰지 않는군요!

마지막 질문은 프로젝트에서 입출력을 파일로 분리하고싶은데 분리하면 더 이해가 어려워질것같다는 질문이었습니다! 너무 불친절하게 질문한것같아요 정말 죄송하고 반성하고있습니다...ㅠㅠ 하지만 이것도 첫번째로 말씀하신것처럼 이런걸 분리하기 이전에 주어진 부분에 충실하게 구현해야하는것 같습니다! 상세한 피드백 정말 감사합니다!